torch/_inductor/codecache.py

```diff
@@ -1727,10 +1727,12 @@ def _compile_consts(consts: bytes, platform: str) -> str:
                         pos += rc
             return consts_o
 
-        from filelock import FileLock
+        from torch.utils.filelock import FileLock
 
         lock_dir = get_lock_dir()
-        lock = FileLock(os.path.join(lock_dir, key + ".lock"), timeout=LOCK_TIMEOUT)
+        lock = FileLock(
+            os.path.join(lock_dir, key + ".lock"), timeout=LOCK_TIMEOUT
+        )
```

Comment:
What's going on with the reformatting here? It makes the PR harder to review

Reply:
I think it's results of me editing the line to WaitCounterFileLock and back.

Reply:
Looks like they're all gone now.